olive/systems/local.py

```diff
@@ -44,7 +44,14 @@ def evaluate_model(
 
         model = model_config.create_model()
         evaluator: OliveEvaluator = OliveEvaluatorFactory.create_evaluator_for_model(model)
-        return evaluator.evaluate(model, data_root, metrics, device=device, execution_providers=execution_providers)
+        return evaluator.evaluate(
+            model,
+            data_root,
+            metrics,
+            device=device,
+            device_id=accelerator.device_id,
```

Comment:
azureml and docker system need handle it as well.

Reply:
azureml do not need this as it run like local system.

Docker needs this one.

Reply:
azureml does have logic to handle accelerator stuff in the entry

Reply:
Yes, somehow, we may need to re-think the part of perf-tuning to let is tune only in one EPs like other passes in the future. Tuning multi-eps were done be Lei before. 

In this way, we can only pass accelerator to evaluator.